backend/.env.dist

```diff
@@ -52,6 +52,9 @@ EMAIL_CODE_REQUEST_TIME=10
 # Webhook
 WEBHOOK_ELOPAGE_SECRET=secret
 
+# EventProtocol
+EVENT_PROTOCOL_ENABLED=true
```

Comment:
Is this really needed? Is there any case where we want to disable the event log?

Reply:
Hmmm, same why log4js can be configured and is not hardcoded... ;-)